public/pikachu.svg

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
At the beginning i was confused about this file, but then i remember the Pikachu from the navbar, cool!